README.md

```diff
@@ -6,17 +6,15 @@
 
 ## People
 
-`actions-runner-controller` is an open-source project currently developed and maintained in collaboration with maintainers @mumoshu and @toast-gear, various [contributors](https://github.com/actions/actions-runner-controller/graphs/contributors), and the [awesome community](https://github.com/actions/actions-runner-controller/discussions), mostly in their spare time.
+`actions-runner-controller` is an open-source project currently developed and maintained in collaboration with the GitHub Actions team, external maintainers @mumoshu and @toast-gear, various [contributors](https://github.com/actions/actions-runner-controller/graphs/contributors), and the [awesome community](https://github.com/actions/actions-runner-controller/discussions).
 
-If you think the project is awesome and it's becoming a basis for your important business, consider [sponsoring us](https://github.com/sponsors/actions-runner-controller)!
+If you think the project is awesome and it's becoming a basis for your important business, please consider sponsoring [external maintainers](https://github.com/sponsors/actions-runner-controller) and specific contributors via GitHub Sponsors.
 
 In case you are already the employer of one of contributors, sponsoring via GitHub Sponsors might not be an option. Just support them in other means!
 
-We don't currently have [any sponsors dedicated to this project yet](https://github.com/sponsors/actions-runner-controller).
+In addition, I(@mumoshu) guess it's also a good idea to subscribe any GitHub plans and/or collaborate with GitHub Actions team members in helpful and friendly manners to show your support to the project. All in all, we want to build an awesome community around this project!
```

Comment:
I'd remove this one as I have plans to update our CONTRIBUTING.md guide with more specific instructions for effective collaboration. WDYT?

Reply:
Awesome!